### DISCUSSION TITLE
Following Issue #15 ...

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/grandorgue#22] [General]
Dear @larspalo 

I allow myself to react here following issue # 15 concerning the reorganization of the menus, and although this issue is closed.
I do not understand your sentence well:

> Unfortunately the comments in oleg68/GrandOrgue#64 show severe lack of understanding on how GrandOrgue works and why things are done the way they are.

Could you explain ? I do not see the connection between reorganizing the menus so that they are more logical and more understandable (elements of parameters of an organ in several menu tabs at the same time for example), and 'a severe lack of understanding on how GrandOrgue works ?

After having discussed it with Oleg, precisely, his final proposal seemed to me to be a really good development for GranOrgue, more clear, logical and precise.

But since English is not my first language, I may have misunderstood what you meant .... :)

**Comment by larspalo** [comment]
In the first comment in the mentioned discussion there was an obvious
confusion about the need for a "Load" and an "Open" menu item. Anyone that
understand GO would know what they are used for (hint: different things).
That is documented in the GO help, by the way. ;-)

**Comment by oleg68** [comment]
@Jean-Gabriel1982 The #15 was related only to the ``Settings...`` menu entry. It has been closed because it has already been done.

We will return to discussing the whole menu structure again after the first GO/github release
